modules/api-svc/src/lib/model/lib/shared.js

```diff
@@ -428,6 +471,61 @@ const internalBulkQuotesResponseToMojaloop = (internal) => {
     return external;
 };
 
+/**
+ * Converts an internal bulk transfer response to mojaloop POST /bulkTransfers/{ID} response
+ *
+ * @returns {object}
+ */
+const internalBulkTransfersResponseToMojaloop = (internal, fulfilments) => {
+    const external = {
+        completedTimestamp: new Date(),
+        bulkTransferState: FSPIOPBulkTransferStateEnum.COMPLETED,
+    };
+
+    if(internal.individualTransferResults && internal.individualTransferResults.length) {
+        const individualTransferResults = internal.individualTransferResults.map((transfer) => {
+            let externalTransfer;
+            // How do we map fsp backend errors to mojaloop errors?
+            // For now we will map any error returned by the backend as a
+            // mojaloop payee rejection and put the fsp backend error into an extension
+            // list.
+            if(transfer.errorResponse) {
+                externalTransfer = {
+                    transferId: transfer.transferId,
+                    errorInformation: Errors.MojaloopApiErrorObjectFromCode(Errors.MojaloopApiErrorCodes.PAYEE_REJECTION).errorInformation,
+                    extensionList: {
+                        extension: [
+                            {
+                                key: 'errorResponseStatusCode',
```

Comment:
I assume this was a mapping that already existed?

Reply:
Not that I'm aware of. All the models that send a backend request to a dfsp are either expecting a successful response or some HTTP error. Theres no in between, the backend api makes little consideration for error messages. 

So in bulk where it's a successful response that contains nested quote/transfer errors I have no precedent of a mapping.

I've had lingering thoughts as to why the sdk doesn't just enforce that the backend it talks to use the FSPIOP api. We are putting all the mappings of a backend to the sdk in a flavoured `core-connector` for dfsp's anyway. 

It's hard to design backend api for a non-existent generic fsp.

Reply:
We already have something like this here --> https://github.com/mojaloop/central-services-error-handling/blob/master/src/factory.js#L147

Reply:
Addressed in https://github.com/mojaloop/sdk-scheme-adapter/pull/404